### PR TITLE
completed js loops practice | Kira Hike

### PR DESCRIPTION
**Pull request by KiraHike**
image

### CODE REVIEW

File: javascript-loops-practice/get-student-names.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* exported getStudentNames */
+
+function getStudentNames(students) {
+  var getStudentNamesResult = [];
+  var object;
+  for (var i = 0; i < students.length; i++) {
+    object = students[i];
+    for (var key in object) {
+      getStudentNamesResult.push(object[key]);
+    }
```
Comment (thebearingedge) [inline javascript-loops-practice/get-student-names.js:10]: No inner loop needed here, as `students[i].name` is what you're after.

File: javascript-loops-practice/filter-out-strings.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* exported filterOutStrings */
+
+function filterOutStrings(values) {
+  var filterOutStringsResult = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] !== typeof '') {
```
Comment (thebearingedge) [inline javascript-loops-practice/filter-out-strings.js:6]: The `typeof` operator returns an exact string value, so you can compare `typeof values[i]` to that.

```js
if (typeof values[i] !== 'string') {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof

**Review by thebearingedge — APPROVED** [review]

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Whoops, please see my previous review
https://github.com/KiraHike/c0121-code-solutions/pull/33#pullrequestreview-579633876

**Comment by KiraHike** [comment]
image
image

**Review by codingcodymiller — APPROVED** [review]
🙌

**Pull request merged**